kubernetes/replicator-gke-cc/docs/index.rst

```diff
@@ -0,0 +1,198 @@
+
+GKE Create Cluster variables
+****************************
+
++--------------------------+---------------+
+| Variable                 | Default       |
++==========================+===============+
+| GKE_BASE_REGION          | us-central1   |
++--------------------------+---------------+
+| GKE_BASE_ZONE            | us-central1-a |
++--------------------------+---------------+
+| GKE_BASE_SUBNET          | default       |
++--------------------------+---------------+
+| GKE_BASE_CLUSTER_VERSION | 1.13.7-gke.24 |
++--------------------------+---------------+
+| GKE_BASE_MACHINE_TYPE    | n1-highmem-2  |
++--------------------------+---------------+
+| GKE_BASE_IMAGE_TYPE      | COS           |
++--------------------------+---------------+
+| GKE_BASE_DISK_TYPE       | pd-standard   |
++--------------------------+---------------+
+| GKE_BASE_DISK_SIZE       | 100           |
++--------------------------+---------------+
+
+This demonstration builds off of the `Confluent Platform on Google Kubernetes Engine demo <https://docs.confluent.io/current/tutorials/examples/kubernetes/gke-base/docs/index.html>`__, you can reference that demo for more information on setting up a base |co-long| deployment on GKE if necessary.
+
+.. _quickstart-demos-operator-replicator-gke-cc-ccloud-setup:
+
+Confluent Cloud Setup
++++++++++++++++++++++
+
+This demonstration requires that you have a |ccloud| account and |ak| cluster ready for use.  See https://www.confluent.io/confluent-cloud/ to get setup with your own account if you do not yet have access.   Once you have your account, see the `Confluent Cloud Quick Start <https://docs.confluent.io/current/quickstart/cloud-quickstart/index.html>`__ to get your first cluster up and running.  If you are creating a new cluster, it is advised to create it within the same Cloud Provider and region as this demo.  This demonstration runs on top of Google Cloud Platform (GCP) and by default in the ``us-central1`` region.
+
+After you have established the |ccloud| cluster you are going to use for the demo, take note of the API Key and Secret clients will use to access the |ccloud| cluster, you will need the values in a momemnt to configure the demo.  See `Create an API Key <https://docs.confluent.io/current/quickstart/cloud-quickstart/index.html#step-4-create-an-api-key>`__ for more details.
+
+To configure the demo to access your |ccloud| account, we are going to create a `Helm Chart values file <https://helm.sh/docs/chart_template_guide/>`__, which the demo looks for in a particular location to pass to ``helm`` commands to weave your cloud account details into the configuration of the |cp| configurations.
+
+Create a values file by executing the following command, first replacing the ``{{ mustache bracket }}`` values for  ``bootstrapEndpoint``, ``username``, and ``password`` with your relevant values.  You can obtain these values from the |ccloud| web console in the CLI & client configuration section.
```

Comment:
1. `You can obtain these...` -- can you find a doc link or include a screenshot with this info?
2. BTW, this can also be done via the CCloud CLI.

Reply:
2. @ybyzek Can you retrieve the secret on the CLI?  All the documentation and CLI commands look to me like you can't grab that

Reply:
@rspurgeon IIUC you can't retrieve a key's secret from an existing key/secret pair, neither through the CLI nor the GUI.  However, I brought up the CLI as a means to create a new key/secret pair (which can also be done through the GUI).